atoma-state/src/state_manager.rs

```diff
@@ -326,12 +335,15 @@ impl StateManager {
         max_num_compute_units: i64,
     ) -> Result<()> {
         sqlx::query(
-            "INSERT INTO node_subscriptions (node_small_id, task_small_id, price_per_compute_unit, max_num_compute_units) VALUES (?, ?, ?, ?)",
+            "INSERT INTO node_subscriptions 
+                (node_small_id, task_small_id, price_per_compute_unit, max_num_compute_units, valid) 
+                VALUES (?, ?, ?, ?, ?)",
         )
             .bind(node_small_id)
             .bind(task_small_id)
             .bind(price_per_compute_unit)
             .bind(max_num_compute_units)
+            .bind(true)
```

Comment:
you can just put `TRUE` in the sql statement above, as you do it in the function bellow

Reply:
Makes a lot of sense actually !